### DISCUSSION TITLE
.NET runtime version check

### DISCUSSION THREAD

**Discussion by rgroenewoudt** [dotnet/runtime#93278] [Q&A] [answered]
I was wondering if there is any way to check if the currently installed .NET runtime is the latest version available?

I know that the current version can be retrieved using `RuntimeInformation.FrameworkDescription` but without winget/chocolately there is no way to check online.
Is there perhaps some online JSON file with the .NET versions?

We would like to warn our system administrators in our software if there is a new .NET version available.

**Comment by MichalPetryka** [comment]
Check releases on https://github.com/dotnet/installer/releases?

**Comment by akoeplinger** [comment, marked as answer]
You can use the release-index.json file at https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json to get the latest version information.

Updates for this file are maintained via https://github.com/dotnet/core/blob/main/release-notes/releases-index.json
